templates/users/details.html.tera

```diff
@@ -2,13 +2,17 @@
 {% import "macros" as macros %}
 
 {% block title %}
-{{ user.display_name }}
+{% if author.display_name %}
```

Comment:
It feels a bit strange to have this `if` in the `title` block, it would be perfect if you moved it outside. Else, everything seems to be working just fine, great job!